scripts/update_and_rebuild_libmesh.sh

```diff
@@ -164,6 +164,7 @@ if [ -z "$go_fast" ]; then
 
   source $SCRIPT_DIR/configure_libmesh.sh
   SRC_DIR=${SCRIPT_DIR}/../libmesh configure_libmesh --prefix=$LIBMESH_DIR \
+                                                     --with-future-timpi-dir=$LIBMESH_DIR \
```

Comment:
```suggestion
```
Place in `configure_libmesh.sh`

Reply:
There is no `LIBMESH_DIR` variable in `configure_libmesh.sh`

Reply:
See my other comment for the equivalent expression.

Reply:
Basically this configure option should mirror `--prefix`, so if `--prefix` can't be moved to `configure_libmesh.sh`, then neither can `--with-future-timpi-dir`

Reply:
Though I see how we're missing generalizing here....166 should be changed to use LIBMESH_DIR instead of SCRIPT_DIR. I will adjust my comment.

Reply:
I see what your intent is here now. Withdrawing my comment, as this is related to where the installed dir will be, not the currently configured one.